alby.go

```diff
@@ -390,6 +390,7 @@ func (svc *AlbyOAuthService) SendPaymentSync(ctx context.Context, senderPubkey,
 			"bolt11":       payReq,
 			"appId":        app.ID,
 			"userId":       app.User.ID,
+			"paymentHash":  responsePayload.PaymentHash,
```

Comment:
```suggestion
			"payment_hash":  responsePayload.PaymentHash,
```

we should move to the underscore default in the logs

Reply:
Ig it's better to do this in a separate PR? Maybe after https://github.com/getAlby/nostr-wallet-connect/pull/134 is also merged?

Reply:
ok